### PR TITLE
Add Okta SDK example

### PR DESCRIPTION
**Pull request by firstmorecoffee**
Example for the Okta integration's is_user_in_group and get_user_info methods.

### CODE REVIEW

File: okta_sdk/impl.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+from sym.sdk.integrations import okta, slack
+from sym.sdk.templates import ApprovalTemplate
+
+
+# Reducers fill in the blanks that your workflow needs in order to run.
+@reducer
+def get_approvers(event):
+    """Route Sym requests to a channel specified in the sym_flow."""
+
+    # engineering team member requests to a separate channel
```
Comment (Arianna2028) [inline okta_sdk/impl.py:11]:
```suggestion
    # Send engineering team member requests to a separate channel
```
?

File: okta_sdk/main.tf [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+  label = "Approval"
+
+  template       = "sym:template:approval:1.0.0"
+  implementation = "${path.module}/impl.py"
+  environment_id = sym_environment.this.id
+
+  vars = {
+    # we will use this in the Flow implementation to check if the approving user
+    # is authorized to approve this request.
+    approvers_group = "00g4fe7l1jPgGOqzA5d7"
```
Comment (Arianna2028) [inline okta_sdk/main.tf:94]: 1. Do we want to make this more clearly an invalid placeholder?
2. Should we link them to [the docs](https://docs.symops.com/docs/okta#add-okta-access-targets) that show how to get the group_id in the comment as well?

File: okta_sdk/main.tf [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+# (e.g. shared integrations or error logging)
+resource "sym_environment" "this" {
+  name            = "main"
+  runtime_id      = sym_runtime.this.id
+  error_logger_id = sym_error_logger.slack.id
+
+  integrations = {
+    slack_id = sym_integration.slack.id
+
+    # Supply our Okta integraiton ID here so we can use `sym.sdk.integrations.okta`
```
Comment (Arianna2028) [inline okta_sdk/main.tf:129]:
```suggestion
    # Supply our Okta integration ID here so we can use `sym.sdk.integrations.okta`
```

**Review by Arianna2028 — APPROVED** [review]
One minor formatting thing, otherwise LGTM

**Review by llam15 — APPROVED** [review]
small comment nit, but otherwise looks great thanks!

**Pull request merged**